src/core/UserIconGroup/UserIconGroup.test.tsx

```diff
@@ -0,0 +1,190 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { render } from '@testing-library/react';
+
+import { UserIconGroup } from './UserIconGroup';
+import UserIcon from '../UserIcon/UserIcon';
+
+it('should render in its most basic state', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium'>
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+    </UserIconGroup>,
+  );
+  expect(container.querySelector('div')).toBeTruthy();
+});
+
+it('should render animated', () => {
+  const { container } = render(
+    <UserIconGroup iconSize='medium' animated={true}>
+      <UserIcon
+        abbreviation='RM'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Robin Mercer'
+      />
+      <UserIcon
+        abbreviation='MR'
+        backgroundColor='#C8C2B4'
+        size='medium'
+        title='Mercer Robin'
+      />
+    </UserIconGroup>,
+  );
+  const userGroupContainer = container.querySelector(
```

Comment:
or just could expect to be truthy element with `.iui-user-icon-list.iui-animated`

Reply:
changed this this suggestion